size_build_configuration.json

```diff
@@ -1,13 +1,14 @@
 {
-  "projectFile": "/googlemac/iPhone/Daylight/sizetracking/testconfigurations/sizetestproject/SizeTest.xcodeproj",
+  "projectFile": "{path-to}/cocoapods-size/sizetestproject/SizeTest.xcodeproj",
```

Comment:
what if you just delete this line? Would it still work?

Reply:
I think it should! Can you please do that thanks :100:

Reply:
Done! :-)